### PR TITLE
New reference directory

### PR DESCRIPTION
**Pull request by flossypurse**
- New reference directory
- Consolidate current references into that directory
- Polish naming
- Update links

### CODE REVIEW

File: docs/system-tools/web-ui.md [hunk 1]
```diff
@@ -148,11 +148,11 @@ The primary mechanism we recommend for debugging is [Temporal Web](https://githu
 
 ![6XkjmR](https://user-images.githubusercontent.com/6764957/110544958-71746480-8167-11eb-8152-8d3a3eb73d4e.gif)
 
-- [Workflows](https://docs.temporal.io/docs/glossary/#workflow) are identified by their [**Workflow ID**](https://docs.temporal.io/docs/glossary/#workflow-id), which you provide when creating the workflow. They also have a **Name** which is directly taken from your code.
+- [Workflows](https://docs.temporal.io/docs/reference/glossary/#workflow) are identified by their [**Workflow ID**](https://docs.temporal.io/docs/reference/glossary/#workflow-id), which you provide when creating the workflow. They also have a **Name** which is directly taken from your code.
```
Comment (djmagee) [inline docs/system-tools/web-ui.md:151]:
```suggestion
- [Workflows](/docs/content/what-is-a-workflow) are identified by their [**Workflow ID**](/docs/content/what-is-a-workflow-id), which you provide when creating the workflow. They also have a **Name** which is directly taken from your code.
```
Should the Workflows link instead be [Workflow Executions](/docs/content/what-is-a-workflow-execution)?

File: docs/system-tools/web-ui.md [hunk 2]
```diff
@@ -148,11 +148,11 @@ The primary mechanism we recommend for debugging is [Temporal Web](https://githu
...
 - Workflow **Status** is usually in one of a few states: Running, Completed, or Terminated, with **Start Time** and **End Time** shown accordingly.
 - Workflow ID's are are distinct from **Run ID's**, which uniquely identify one of potentially many Runs of Workflows with the same Workflow ID.
 
-> Tip: Don't confuse Runs with [Workflow Executions](https://docs.temporal.io/docs/glossary/#workflow-execution) - they are similar, but a long-running Workflow Execution can have multiple Runs. A Run is the atomic unit.
+> Tip: Don't confuse Runs with [Workflow Executions](https://docs.temporal.io/docs/reference/glossary/#workflow-execution) - they are similar, but a long-running Workflow Execution can have multiple Runs. A Run is the atomic unit.
```
Comment (djmagee) [inline docs/system-tools/web-ui.md:155]:
```suggestion
> Tip: Don't confuse Runs with [Workflow Executions](/docs/content/what-is-a-workflow-execution)—they are similar, but a long-running Workflow Execution can have multiple Runs. A Run is the atomic unit.
```

**Review by djmagee — APPROVED** [review]
Cutting out the middleman for some of the links...

**Comment by flossypurse** [comment]
> Cutting out the middleman for some of the links...

Ya, I totally noticed that and was going to update. Good catch!

File: docs/system-tools/web-ui.md (hunk 1 above)
Comment (flossypurse) [reply docs/system-tools/web-ui.md:151]: Ya, this whole paragraph is full of ambiguity.

File: docs/system-tools/web-ui.md (hunk 2 above)
Comment (flossypurse) [reply docs/system-tools/web-ui.md:155]: Oh man, I hadn't noticed this before, but this message is super contradictory.

**Pull request merged**